imap/jmap_mailbox.c

```diff
@@ -750,7 +770,7 @@ static int jmap_mailbox_get_cb(const mbentry_t *mbentry, void *_rock)
 
     /* Move this mailbox of the lookup list */
     if (rock->want) {
-        hash_del(mbentry->uniqueid, rock->want);
+        hash_del(id, rock->want);
```

Comment:
This fails to compile with `-Og`:
```
imap/jmap_mailbox.c: In function 'jmap_mailbox_get_cb':
imap/jmap_mailbox.c:773:9: error: 'id' may be used uninitialized [-Werror=maybe-uninitialized]
  773 |         hash_del(id, rock->want);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~
imap/jmap_mailbox.c:740:17: note: 'id' was declared here
  740 |     const char *id;
```

Probably just `const char *id = NULL;` ...

Reply:
Fixed